protocol/app/ante/sigverify_test.go

```diff
@@ -33,30 +61,42 @@ func TestSigVerification(t *testing.T) {
 
 	addrs := []sdk.AccAddress{addr1, addr2, addr3}
 
+	msgs := make([]sdk.Msg, len(addrs))
+	accs := make([]sdk.AccountI, len(addrs))
 	// set accounts and create msg for each address
 	for i, addr := range addrs {
 		acc := suite.AccountKeeper.NewAccountWithAddress(suite.Ctx, addr)
-		require.NoError(t, acc.SetAccountNumber(uint64(i)))
+		require.NoError(t, acc.SetAccountNumber(uint64(i)+1000))
 		suite.AccountKeeper.SetAccount(suite.Ctx, acc)
+		msgs[i] = testdata.NewTestMsg(addr)
+		accs[i] = acc
 	}
 
 	feeAmount := testdata.NewTestFeeAmount()
 	gasLimit := testdata.NewTestGasLimit()
 
 	spkd := sdkante.NewSetPubKeyDecorator(suite.AccountKeeper)
-	svd := customante.NewSigVerificationDecorator(
-		suite.AccountKeeper,
-		suite.ClientCtx.TxConfig.SignModeHandler(),
+	txConfigOpts = authtx.ConfigOptions{
```

Comment:
I tried to minimize the number of changes to simplify how much work is needed when porting over the code from Cosmos SDK so that diffing the original and the forked version is simpler next time when porting over the code.

I wish there was a way to not have to have copies of this code/tests since it is cumbersome to maintain.